src/vm/jitinterface.cpp

```diff
@@ -9953,8 +9953,8 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
 
 /*********************************************************************/
     // return address of fixup area for late-bound N/Direct calls.
-void* CEEInfo::getAddressOfPInvokeFixup(CORINFO_METHOD_HANDLE method,
-                                        void **ppIndirection)
```

Comment:
I think a better name for this method would be `getAddressOfPInvokeTarget` since the result may or may not be a fixup now.

Also, you can rename `getPInvokeUnmanagedTarget` to something like `getPInvokeUnmanagedTarget_Obsolete` while you are on it. In the ideal case, it would be deleted but I am not sure whether it is possible because of full .NET framework compat.